test/e2e/network_test.go

```diff
@@ -322,10 +316,23 @@ func (tc *testContext) getPodIP(selector metav1.LabelSelector) (string, error) {
 	return podList.Items[0].Status.PodIP, nil
 }
 
+// getRelevantWindowsServerContainerImage gets the appropriate WindowsServer image based on VXLAN port
+func (tc *testContext) getRelevantWindowsServerContainerImage() string {
```

Comment:
`getWindowsServerContainerImage()` will suffice. Using terms like `Relevenat` `Valid` sort of implies we have a use case where we need to get irrelevant or invalid entities.